engine-comparison/dispatcher.sh

```diff
@@ -66,6 +66,17 @@ build_benchmark_using() {
   mkdir $SEND_DIR
 
   cp ${BUILDING_DIR}/${BENCHMARK}-${FUZZING_ENGINE} $SEND_DIR
+
+  #TODO: make these gsutil cp?
+  cp $WORK/FTS/engine-comparison/Dockerfile $SEND_DIR
+  cp $WORK/FTS/engine-comparison/runner.sh $SEND_DIR
+  cp $WORK/FTS/engine-comparison/config/parameters.cfg $SEND_DIR
+  cp $FENGINE_CONFIG $SEND_DIR/fengine.cfg
+
+  echo "BENCHMARK=$BENCHMARK" > ${SEND_DIR}/metadata.cfg
+  echo "FUZZING_ENGINE=$FUZZING_ENGINE" >> ${SEND_DIR}/metadata.cfg
+
+  # TODO: ensure all seeds are in $BENCHMARK/seed
```

Comment:
This would be something to ensure in every individual `build.sh`, so its more of a long-term todo